### PR TITLE
9 check model

### PR DESCRIPTION
**Pull request by nkoblai**

### CODE REVIEW

File: moninag/check/models.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+# Create your models here.
```
Comment (lhalam) [inline moninag/check/models.py]: delete this comment

File: moninag/check/admin.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (lhalam) [inline moninag/check/admin.py]: delete this comment

File: moninag/check/views.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```
Comment (lhalam) [inline moninag/check/views.py]: delete this comment

File: moninag/check/models.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+class Check(models.Model):
+
```
Comment (lhalam) [inline moninag/check/models.py:7]: add docString

File: moninag/check/models.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+	name = models.CharField(max_length=20)
+	plugin_name = models.CharField(max_length=20)
+	run_freq = models.IntegerField()
```
Comment (lhalam) [inline moninag/check/models.py]: add a default value

File: moninag/check/tests.py [hunk 6]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (lhalam) [inline moninag/check/tests.py]: delete this comment

File: moninag/check/models.py [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+	def __str__(self):
+		return "Check Name: %s, Plugin Name: %s, Run Frequency: %d".format(self.name,
+                                                    					   self.plugin_name,
+                                                    					   self.run_freq)
```
Comment (lhalam) [inline moninag/check/models.py]: correct formatting space
add end line

**Review by byshovets — APPROVED** [review]

**Pull request merged**